src/main/java/com/google/sps/classes/Utility.java

```diff
@@ -16,7 +16,9 @@
 
 import com.google.appengine.api.users.UserService;
 import com.google.appengine.api.users.UserServiceFactory;
+import com.google.sps.classes.SqlConstants;
 import com.google.gson.Gson;
+import com.google.sps.classes.SqlConstants;
```

Comment:
It looks like you're including this twice?

Reply:
Whoops, removed!